core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.controller.spec.ts

```diff
@@ -13,53 +13,180 @@
 // limitations under the License.
 
 /**
- * @fileoverview Unit tests for NewChapterTitleModalController.
+ * @fileoverview Unit tests for CreateNewChapterModalController.
  */
 
-describe('New Chapter Title Modal Controller', function() {
+import { UpgradedServices } from 'services/UpgradedServices';
+
+describe('Create New Chapter Modal Controller', function() {
   var $scope = null;
+  var $q = null;
+  var $rootScope = null;
   var $uibModalInstance = null;
+  var StoryUpdateService = null;
+  var StoryEditorStateService = null;
+  var StoryContentsObjectFactory = null;
+  var StoryObjectFactory = null;
+  var ExplorationIdValidationService = null;
   var nodeTitles = ['title 1', 'title 2', 'title 3'];
 
   beforeEach(angular.mock.module('oppia'));
+
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+      $provide.value(key, value);
+    }
+  }));
```

Comment:
Should we use TestBed to inject dependencies instead?

Reply:
@srijanreddy98 can help you out. But this needs to be fixed.

Reply:
https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#providing-angular-8-services-in-downgrade-files-when-it-uses-any-upgraded-service-as-a-dependency

Reply:
I am fine with ugs being used for now (if it is urgent). Create an issue and assign that to yourself and me.

Reply:
Sure, thanks a lot. @srijanreddy98

Reply:
This has been fixed(dont need to open an issue). Done.